src/java/org/apache/cassandra/tools/StandaloneScrubber.java

```diff
@@ -112,56 +112,58 @@ public static void main(String args[])
             System.out.println(String.format("Pre-scrub sstables snapshotted into snapshot %s", snapshotName));
 
             // if old-style json manifest, snapshot it
-            if (cfs.directories.tryGetLeveledManifest() != null)
-            {
+            if (cfs.directories.tryGetLeveledManifest() != null) {
                 cfs.directories.snapshotLeveledManifest(snapshotName);
                 System.out.println(String.format("Leveled manifest snapshotted into snapshot %s", snapshotName));
             }
 
-            LeveledManifest manifest = null;
```

Comment:
It happens outside the loop, but needs to be visible to the Runnable<>. So it either has to be a member variable or a final variable. Making it final would've required converting the if statement that follows its declaration into a function, so it seemed simpler to just promote manifest to be a member variable.